### DISCUSSION TITLE
speedify_cli monitoring

### DISCUSSION THREAD

**Discussion by zbynekdrlik** [SmoothWAN/SmoothWAN#176] [Ideas and feature requests]
Hi, is it possible  to show some monitoring on console similar as it is on web access?
Is sppedify_cli somehow accessibly from console?

**Comment by ClintDavis** [comment]
speedify_cli is there:
/usr/share/speedify/speedify_cli

There are no current plans for speedify to add any kind of snmp or external graphing solution. However, the speedify cli allows you to pull some traffic stats through the cli, but you would need to periodically pull, format and place into some RRD visualiser system, cacti / grafana etc.

I've thought about doing something similar, however the awesome thing about Smoothwan is pulling through the app output right onto the website dashboard over HTTP, so solved my immediate requirement.

**Comment by TalalMash** [comment]
Adding a simple TUI wrapper next release.

**Comment by zbynekdrlik** [comment]
Sounds so great :)
